backend/src/utils/cors.js

```diff
@@ -18,7 +18,7 @@ exports.cors = function(req, res, next) {
 }
 
 exports.corsProd = function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', 'https://smartcourse.me')
+    res.header('Access-Control-Allow-Origin', 'https://smartcourse.me, https://wwww.smartcourse.me')
```

Comment:
I'm fairly sure you can't specify multiple origin in this header. The only way to deal with this is to dynamically update the header at runtime depending on the request.

Reply:
Yeah you are right, will fix this.